frontend/src/pages/signin-page.tsx

```diff
@@ -0,0 +1,109 @@
+import { Link, useNavigate } from 'react-router-dom';
+import AddGoogleIcon from '@/assets/svg/google-color-icon.svg';
+import AddGithubIcon from '@/assets/svg/github-icon.svg';
+import { useForm } from 'react-hook-form';
+import type { FieldValues } from 'react-hook-form';
+import { TSignInSchema, signInSchema } from '@/lib/types';
+import 'react-toastify/dist/ReactToastify.css';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { toast } from 'react-toastify';
+
+function signin() {
+  const navigate = useNavigate();
+
+  const { register, 
+        handleSubmit, 
+        formState: { errors , isSubmitting},
+        reset
+      } = useForm<TSignInSchema>({resolver: zodResolver(signInSchema)});
+
+  const onSubmit = async (data: FieldValues) => {
+
+    if(data.email === "abc@gamil.com"){
+      toast.error('Submitting form is failed');
+      return;
+    }
+    
+    // TODO: Server-side validation
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+    
+    reset();
+    navigate("/");
+  }
+
+  return (
+    <div className="flex-grow cursor-default bg-white px-5 py-9 mt-1">
+      <div className="mb-4 flex justify-center">
+        <div className="flex items-center justify-center space-x-4 w-5/6 md:w-5/6 lg:w-2/5">
+          <h2 className="text-center text-2xl sm:text-3xl md:text-4xl lg:text-2xl xl:text-3xl 2xl:text-4xl font-bold text-black w-3/4">
+            Sign in to WanderLust
```

Comment:
Please don't use anything other than plain classnames and sm: classnames, for mobile the default,  without any prefix .... and for screen more than tablet sm: will work. so kindly remove xl: md: lg: anywhere used

Reply:
Got it.

Reply:
Done.